sklearn/externals/array_api_extra/_lib/_funcs.py

```diff
@@ -822,7 +822,7 @@ def nunique(x: Array, /, *, xp: ModuleType | None = None) -> Array:
 
     # xp does not have unique_counts; O(n*logn) complexity
     x = xp.reshape(x, (-1,))
-    x = xp.sort(x)
```

Comment:
We should not edit the content of `sklearn/externals/array_api_extra/_lib/_funcs.py` in a scikit-learn level PR but rather contribute this change upstream to the `array-api-extra` project. Then once the contribution is accepted and released we can update our vendored version of `array-api-extra` under `sklearn/externals`.